test/e2e/kubernetes/node/node.go

```diff
@@ -74,14 +74,17 @@ type List struct {
 // AreAllReady returns a bool depending on cluster state
 func AreAllReady(nodeCount int) bool {
```

Comment:
should this function return "true" if there is an extra not-ready node? If not, we should return false when we find a node in "NotReady" state

Reply:
This function should only return true when the exact desired number of nodes are all ready (so, an extra not ready means false).